### PR TITLE
Add Danfysiks 10 and 11

### PR DESCRIPTION
**Pull request by AdrianPotter**
### Description of work

Add Danfysiks 10 and 11

### To test

https://github.com/ISISComputingGroup/IBEX/issues/1512

### Acceptance criteria

Danfysiks 10 and 11 exist and can be controlled as in 1-9

---

#### Code Review

- [x] Is the code of an acceptable quality?
- [x] Are the PVs named according to the [naming standards](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming).
- [x] Have suitable [disable records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Disable-records) been added?
- [x] Have suitable [simulation records](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Record-Simulation) been added?
- [x] Have the [finishing touches](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/IOC-Finishing-Touches) been applied?
- [ ] Is there an [emulator](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Emulating-Devices) for the device?
- [x] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev). If so, do they describe the changes appropriately?

### Functional Tests

- [x] Do changes function as described? Add comments below that describe the tests performed.
- [x] Does the IOC respond correctly both in full and simulation mode, where it's possible to test both?
- [ ] If there are multiple _0n IOCs, do they run correctly?

### Final steps

- [ ] Reviewer has updated the submodule in the main EPICS repo? See **Reviewing work for the subModules of EPICS** in the [Git workflow](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Git-workflow) page for details.
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

File: DFKPS/iocBoot/iocDFKPS-IOC-10/st.cmd [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Register all support components
+dbLoadDatabase "dbd/DFKPS-IOC-10.dbd"
+DFKPS_IOC_10_registerRecordDeviceDriver pdbbase
+
+## Load FileList
+## A seperate instance must be created for each danfysik
+epicsEnvSet "RAMP_DIR" "C:/Instrument/Settings"
+epicsEnvSet "RAMP_PAT" ".*"
+FileListConfigure("RAMPFILELIST9", $(RAMP_DIR), $(RAMP_PAT))
```
Comment (ThomasLohnert) [inline DFKPS/iocBoot/iocDFKPS-IOC-10/st.cmd]: This should be RAMPFILELIST10

File: DFKPS/iocBoot/iocDFKPS-IOC-11/st.cmd [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Register all support components
+dbLoadDatabase "dbd/DFKPS-IOC-11.dbd"
+DFKPS_IOC_11_registerRecordDeviceDriver pdbbase
+
+## Load FileList
+## A seperate instance must be created for each danfysik
+epicsEnvSet "RAMP_DIR" "C:/Instrument/Settings"
+epicsEnvSet "RAMP_PAT" ".*"
+FileListConfigure("RAMPFILELIST9", $(RAMP_DIR), $(RAMP_PAT))
```
Comment (ThomasLohnert) [inline DFKPS/iocBoot/iocDFKPS-IOC-11/st.cmd]: This should be RAMPFILELIST11

**Comment by ThomasLohnert** [comment]
Everything looks good, just forgot to change the IOC number for Rampfilelist.

File: DFKPS/iocBoot/iocDFKPS-IOC-11/st.cmd (hunk 2 above)
Comment (AdrianPotter) [inline DFKPS/iocBoot/iocDFKPS-IOC-11/st.cmd]: Well spotted, thanks!

**Pull request merged**